backend/src/Edna/Config/Definition.hs

```diff
@@ -87,7 +87,7 @@ instance Aeson.FromJSON LoggingConfig where
     let value = parseLoggingConfig Nothing $ T.unpack text
     in case value of
       Right x -> return x
-      Left _  -> fail "Failed to parse LoggingConfig option"
+      Left _  -> error $ "Invalid config value: " <> show text
```

Comment:
`fail` is better. `error` is ok for programmer mistakes – code paths that are expected to never be executed regardless of external input. This `FromJSON` instance is used to parse application config that is provided by a user, so we can't be sure that error will never happen. Users may supply malformed values in the config and `fail` is better in this case, because with `fail` it will be more predictable when the error will happen.

Reply:
I've found that we have another similar definition with `error`, so I've changed it here to use the same approach. I'll remember that `fail` is better and change it